docs/Compile.rst

```diff
@@ -463,40 +463,102 @@ Building
 
 Windows
 =======
-On Windows, DFHack replaces the SDL library distributed with DF.
+.. contents:: Section Contents
+  :local:
+  :depth: 3
+
+In order to build on windows you'll need to resolve a few dependencies. The dfhack build system uses perl to generate
+source files using xml files detailing various DF structures. Cmake is used to generate build files, or project files
+for visual studio. Read the sections below for more detail.
+
+Note: On Windows, DFHack replaces the SDL library distributed with DF.
 
 Dependencies
 ------------
-You will need the following:
+These are the general dependencies you will need:
 
 * Microsoft Visual C++ 2022, 2019, 2017, or 2015 (optional)
 * Microsoft Visual C++ 2015 Build Tools
 * Git
 * CMake
-* Perl with XML::LibXML and XML::LibXSLT
-
-  * It is recommended to install StrawberryPerl, which includes both.
+* StrawberryPerl
```

Comment:
I think the previous phrasing was better. Another Perl will work. And in the new version, it's not clear that StrawberryPerl includes LibXML/LibXSLT already, which could lead people on a path to install those manually (unless StrawberryPerl does _not_ include those libraries?)

Reply:
I can respect that, I didn't like the old phrasing because I thought it was giving too many details they didn't need. (Windows User: ok what do I install? A: StrawberryPerl)

Maybe I could add some extra detail to get us back to an equivalent level of detail? While still starting off with what they need to install, and then showing them individual dependencies.

Reply:
How about having a sub-bullet read:

```
    * Note: if you install a different version of Perl, make sure you also install the ``XML::LibXML`` and ``XML::LibXSLT`` packages
````

also worth noting that it is `::` and not `:` in the package names

Reply:
Oops